timescaledb/getting-started/create-cagg/create-cagg-policy.md

```diff
@@ -0,0 +1,75 @@
+# Using and setting up continuous aggregate policies
+
+You now have a continuous aggregate, but you have not updated any data or created an automatic 
+policy. There are two ways to update a continuous aggregate: 
+1. Manual refresh
+2. Automatic continuous aggregate refresh policy
```

Comment:
In the compression docs, we did this in reverse order: Automatic first and then manual.

IMO that's a good order because we want to encourage people to setup policies over manual usage. Totally fine to show both, but I think we should show them how to setup up in our preferred way first.